src/types/config.ts

```diff
@@ -0,0 +1,29 @@
+import { TheGraphApiImplParams } from 'api/thegraph/TheGraphApi'
+import { MultiTcrApiParams } from 'api/tcr/MultiTcrApi'
+import { DexPriceEstimatorParams } from 'api/dexPriceEstimator/DexPriceEstimatorApi'
+
+export interface MultiTcrConfig {
+  type: 'multi-tcr'
+  config: Omit<MultiTcrApiParams, 'web3'>
+}
+
+export interface NoTcrConfig {
+  type: 'none'
+}
+
+export interface DexPriceEstimatorConfig {
+  type: 'dex-price-estimator'
+  config: DexPriceEstimatorParams
+}
+
+export interface TheGraphApiConfig {
+  type: 'the-graph'
+  config: TheGraphApiImplParams
+}
+
+export interface Config {
+  // name: string
```

Comment:
No needed in the loaded app I guess. Remove?

Reply:
It is needed, I would make it mandatory. Not sure how it works without this

Reply:
I've just made it mandatory and added a test for it